nodejs/week2/app.js

```diff
@@ -0,0 +1,76 @@
+import express from 'express';
+import fs from 'fs';
+
+const app = express();
+const port = process.env.PORT || 3000;
+
+app.use(express.json());
+
+const documents = JSON.parse(fs.readFileSync('documents.json', 'utf8'));
```

Comment:
this works. You can alternatively use `import data from "./documents.json" assert { type: "json" };`